### ISSUE TITLE
Both UK & US English note values are reported when set to UK English

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31479]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1.  In the Preference General settings select US English.
2. View any score with a variety of note values and notice as you click on them the status reports them as whole, half, quarter, eighth, etc.  This is correct behaviour.
3. Change the language in preferences to UK English.
4. Click on the same notes and notice you see in the status semibreve whole, minim half, crotchet, quaver eighth.  Except for the crotchet all are reported in both English UK and English US.

This is primarily an issue for screen readers as we are hearing both.  This added speech slows down further what is already a slow process of having the status read out to identify each note, it also creates additional mental overload processing each string of info.  If it is simple to update translation tables could this be quickly fixed as it has been on a feature request I've just closed for over 2 years.  

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.4

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by Jojo-Schmitz** [comment]
Ad step 4: in the status bar you'd see "Semibreve (whole)", "Minim (half)", "Crotchet" and "Quaver (eighth)" (notice capitalization and parentheses). And that is because the en_GB translation has them translated that way. the missing " (quarter)" for "Crotchet" is an ommision and inconsistency in the translations, which I've fixed just now.

It could of course get changed into the other direction: removing those " (whole)", " (half)", " (quarter)" and " (eighth)" strings from the translations. Either all of them, or just those used in the status bar.
But I mentioned this and its shortcommings in https://github.com/musescore/MuseScore/issues/20052#issuecomment-1816016772 already

There are some more of those: "Semiquaver (16th)", "Demisemiquaver (32nd)" and "Hemidemisemiquaver (64th)". Not for 128th or shorter though...

As far as I can tell it is the strings 2739-2740 and 2745-2749 on Transifex, which are used for the status bar and )as such) also for accessibilty (screen readers)

<img width="734" height="791" alt="Image" src="https://github.com/user-attachments/assets/64e5cd7c-ea06-479d-88a4-9e5a76d8ea96" />

**Comment by jmwmusic** [comment]
Just to be clear, the bug here is that whole, half, quarter, etc. for all of these should *NOT* be read out by the screen reader when UK English is selected.  So the crotchet was the only one correct currently.  Speaking the note values twice is the bug.

**Comment by Jojo-Schmitz** [comment]
To be clear: it is not a bug, but just how the translations work, so it is by design, and esp. this is not a bug in MuseScore Studio's code, which basically is what this issue tracker is about. 
The duration is **not** read out twice (e.g. "quaver quaver"), but in two different forms (e.g. "quaver 8th"), and this is done for clarity. And it is the same strings that are used for the status bar and the screen readers, so no way to differentiate between them depending on where they are getting used.
We could change those 7 strings shown above in the en_GB translation, but that'd change them for screen readers **and** status bar

**Comment by MarcSabatella** [comment]
I think changing the translation is a reasonable thing to consider.  I'm not sure the parenthesized US English names really add value for people using the UK translation.

**Comment by Jojo-Schmitz** [comment]
Maybe. But a decision to be made by British English speakers

**Comment by jmwmusic** [comment]
I am a British English speaker as are any others who will have selected this as our language option.  so am still unclear why we would also be given the US translations.  If you select US as your language the minim, crotchet, etc are not put in brackets after that!  In New Zealand, Australia, Canada, UK (and probably all other non US English areas) we have had in our exams, schools syllabus and method books for learning music only the UK english terms.  Only very recently to aid global communication the US terms have been added, but would still not be used for example in an exam paper or usual speech.

This may be by design, but for accessibility it is not good design to have something stated twice, even if slightly differently.  This slows down what is already a very slow process to wait to hear all the details for each note and adds to mental strain trying to take in all the info.

**Comment by Jojo-Schmitz** [comment]
I'd rather await @shoogle 's view on this, as he's a) a British English speaker and b) our resident accessibility guru ;-)

**Comment by jrbowden** [comment]
Hi, I have noted this problem for years. I am a British speaker and live in UK, and use a screen reader. I agree with @jmwmusic: having to hear both the UK and US names is not good.
I expect to hear, "semibreve", "minim", "crotchet", "quaver" (etc).
Thank you for considering updating the translation file.

**Comment by shoogle** [comment]
I agree it's a waste of time to give both the British and the American name for every note in the score.

<img width="789" height="162" alt="Status bar text: Note; Pitch C4; Duration: Crotchet (quarter);" src="https://github.com/user-attachments/assets/b7982300-a521-46ef-bb2c-80806dcde039" />

<br>We should only give **one name** in the score. This applies to both screen reader speech and status bar text.

However, for duration buttons in the note-input toolbar, I think there's educational value in showing both names. This way the user can go to the toolbar if they want to find out what the other name for a particular duration is. It would appear in the tooltip as well as in screen reader speech for that button.

<img width="708" height="176" alt="Note-input toolbar: Crotchet (quarter) note button tooltip" src="https://github.com/user-attachments/assets/f429426f-c226-4a9e-82f8-82ccf4a35068" />


<br>We could show this even when the language is set to US English:

* English (GB) users would see _"Crotchet (quarter)"_.
* English (US) users would see _"Quarter (crotchet)"_.

Again, _only in the note-input toolbar_; elsewhere you'd just see your native note names.

And for durations of 128th and smaller, all English users would only ever see the American name, as is currently the case.

<img width="278" alt="All durations in the Customize toolbar menu for the note-input" src="https://github.com/user-attachments/assets/de99b828-4961-45b5-b619-1ee30b28927d" />

**Comment by Jojo-Schmitz** [comment]
OK, en_GB and en_US translations fixed. Update your translations (in some 15 minutes), check and then close this issue

**Comment by jmwmusic** [comment]
Brilliant, I will let the accessibility group know how to update.  I didn't know the check for language updates button was there as like in so many dialogues Tab skips it!  But also, it doesn't give any announcement that it is downloading so I didn't think the button was working.  I'll report that one separately.  I had to call someone to look at it for me.
